meta-ostro/recipes-swupd/swupd-client/oe-swupd-helpers.bbappend

```diff
@@ -1,3 +0,0 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
-
-SRC_URI_append = "file://efi-combo-trigger.service"
```

Comment:
Previously meta-swupd's oe-swupd-helpers.bb was responsible for installing the file `efi-combo-trigger.service`. Now nothings install it onto the target. As result `update-triggers.service` doesn't actually run efi-combo-updater.